### DISCUSSION TITLE
create repository failed, unable to unmarshall repository json: unexpected end of JSON input

### DISCUSSION THREAD

**Discussion by jimmi4u** [fluxcd/flux2#3454] [Q&A]
Hi, I'm trying to bootstrap with:

export BITBUCKET_TOKEN=my_token

flux bootstrap bitbucket-server \   
  --components-extra=image-reflector-controller,image-automation-controller  \
  --token-auth \
  --hostname=bitbucket.org \
  --owner=project_name \
  --username=username \
  --repository=exiisting_repo \
  --branch=master \
  --path=cluster/master \
  --personal 

but got:
failed to create new Git repository "https://bitbucket.org/project_name/existing_repo": failed to create repository ~project_name/existing_repo: failed to create repository: create repository failed, unable to unmarshall repository json: unexpected end of JSON input

Has anybody a hint on how to solve this?

Thanks a lot!

**Comment by martin0598** [comment]
Hi jimmy4u,
got the same issue, did you find a solution?

best regards,
Martin
